Src/DependencyCollector/Microsoft.AI.DependencyCollector.Tests/InstrumentationKeyHashLookupHelperTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
```

Comment:
There is no License.txt file in the repo.
Our NuGet packages link to here: [http://go.microsoft.com/fwlink/?LinkID=510709](http://go.microsoft.com/fwlink/?LinkID=510709)